src/main/scala/managehelpcontentpublisher/PathAndContent.scala

```diff
@@ -91,12 +94,16 @@ object PathAndContent {
         content <- newContent
       } yield storeTopic(PathAndContent(moreTopics.path, content))
 
-      if (articleTopics.forall(topic => config.topic.corePaths.contains(topic.path))) Right(None)
+      // No need to do anything if the new topics and the topics of the old article are all core topics
+      if (
+        newTopics.forall(topic => config.topic.corePaths.contains(topic.path)) &&
+        oldArticle.forall(_.topics.forall(topic => config.topic.corePaths.contains(topic.path)))
```

Comment:
Maybe factor out the predicate give it is duplicated, something like so

```
val isCoreTopic = (topic: Topic) => config.topic.corePaths.contains(topic.path)
...
if (newTopics.forall(isCoreTopic) && oldArticle.forall(_.topics.forall(isCoreTopic)) { ...
```

Reply:
It's a good suggestion but they're actually slightly different types that both have a String `path` property.  So I've improved it a little in 994b504 without changing the types.